Mathlib/Algebra/Divisibility/Hom.lean

```diff
@@ -26,9 +26,3 @@ variable {M N : Type*}
 theorem map_dvd [Semigroup M] [Semigroup N] {F : Type*} [FunLike F M N] [MulHomClass F M N]
     (f : F) {a b} : a ∣ b → f a ∣ f b
   | ⟨c, h⟩ => ⟨f c, h.symm ▸ map_mul f a c⟩
-
-theorem MulHom.map_dvd [Semigroup M] [Semigroup N] (f : M →ₙ* N) {a b} : a ∣ b → f a ∣ f b :=
```

Comment:
Deleting these `.map_dvd` lemmas might be a good idea, but please do so in a separate PR. (And, as below, please add deprecations.)